CMakeLists.txt

```diff
@@ -290,6 +290,9 @@ CMAKE_DEPENDENT_OPTION(
   "Slicer_BUILD_ITKPython" OFF)
 mark_as_superbuild(Slicer_INSTALL_ITKPython)
 
+option(Slicer_BUILD_PARAMETERSERIALIZER_SUPPORT "Build Slicer with parameter serializer support" OFF)
```

Comment:
Certainly the SlicerExecutionModel should not depend on VTK. We are talking about how Slicer modules will access jsoncpp and how to make sure the same jsoncpp version is used by SlicerExecutionModel, VTK, and Slicer modules. One solution could be that Slicer superbuild builds jsoncpp and passes it to both SlicerExecutionModel and VTK.